README.md

```diff
@@ -1,28 +1,25 @@
 
-Welcome to the SONiC NAS Host Adapter.
+Welcome to the SONiC NAS Host Adapter
 ======================================
-This SONiC repo contains the manifest file for the repo tool used to pull down sources for the SONiC NAS Project.
+This SONiC repository contains the manifest file for the repo tool used to pull down sources for the SONiC NAS Project. The SONiC NAS project is the SAI host adapter originally written by Dell and contributed to the SONiC project. It is assumed that the user is familiar with Linux and has some basic development knowledge.   
 
-The SONiC NAS project is the SAI host adapter originally written by Dell contributed to the SONiC project.
-
-It is assumed that the user is familiar with Linux and has some basic development knowledge.   
-
-
-Reading the documentation
+Documentation
 -------------------------
-Comprehensive documentation is available at the following link: [SONiC NAS Documentation](http://confluence.force10networks.com/pages/viewpage.action?spaceKey=OPS&title=OS10%20Open%20Source%20Documentation)
+Comprehensive information is available at [SONiC NAS Documentation](https://github.com/amybuck/sonic-nas-manifest)
```

Comment:
I think you want the link to be:

https://github.com/azure/sonic-nas-manifest

This way it doesn't point to your personal fork